app.js

```diff
@@ -10,6 +31,141 @@ function Book(title, author, pages, read) {
     }
 }
 
-myBook = new Book('Lord of the Rings', 'Tolkien', 365, true);
+function createBook(book) {
+  myLibrary.push(book);
+  document.querySelector('.input-form').classList.toggle('form-visibility');
+  localStorage.setItem("myLibrary", JSON.stringify(myLibrary));
+  render(myLibrary);
+}
+
+// Reads input values from the form.
+function readInputValues(){
+  return {
+  bookTitle: document.getElementById("book-title").value,
+  authorName: document.getElementById("author-name").value,
+  description: document.getElementById("book-description").value,
+  pagesNum: document.getElementById("book-pages").value,
+  status: document.querySelector('input[name="read"]:checked').value
+  }
+}
+function addBookToLibrary() {
+  let bookInfo = readInputValues();
+  const id = randomId();
+  let book =new Book(id, bookInfo.bookTitle, bookInfo.authorName, bookInfo.description, bookInfo.pagesNum, bookInfo.status);
+  let valid = inputValid(bookInfo.bookTitle, bookInfo.authorName, bookInfo.description, bookInfo.pagesNum);
+
+  if (valid == true) {createBook(book);}
+  clearForm();
+}
+
+
+function clearForm()
+{
+  form.reset();
+  operation = null;
+}
+// function for handling click event on New Form.
+toggleForm(formBtn);
+
+function toggleForm(button) {
+  button.addEventListener('click', function(){
+    form.classList.toggle('form-visibility');
+  });
+}
+
+// function for handling click event on Submit.
+submitBook.addEventListener("click", selectOperation);
+
+  function render(arr){
+    let html = "";
+    for (let book of arr) {
+      let readingStatus = book.read == "read" ? "You have read this book" : "You haven't read this book";
+      html += `<div class= "book-card"><h3 class="book-title">${book.title}</h3>
+              <p class="author-name">by ${book.author}</p>
+              <p class="description">${book.description}</p>
+              <div class="book-details">
+                <p class="book-pages"><i class="fa fa-chevron-right"></i> No. of Pages: <span>${book.pages}</span></p>
+                  <p><i class="fa fa-chevron-right"></i> ${readingStatus}</p>
+              </div>
+              <div class="btns">
+                <button class="btn-read-status book-buttons" onclick="changeStatus('${book.id}')">Read</button>
+                <button class="btn-remove book-buttons" onclick="deleteBook('${book.id}')">Delete</button>
+                <button class="btn-edit book-buttons" onclick="editBookInfo('${book.id}')">Edit</button>
+              </div>
```

Comment:
If after you configure your linting properly you keep on getting the unused error for those onclick props it is ok to rule it out of the eslintrc.json file.